locksmith/__tests__/controllers/metadataController/requestingKeyholderMetadata.test.ts

```diff
@@ -127,7 +127,7 @@ describe('Metadata Controller', () => {
         )
 
         const response = await request(app)
-          .get(`/api/key/${lockAddress}/keyHolderMetadata`)
+          .get(`/api/key/${lockAddress}/keyHolderMetadata?chain=${chain}`)
```

Comment:
Is this relevant to this PR?

Reply:
yes, test will fail, because req.chain should be 31337

Reply:
ok